### PR TITLE
st1 [1985][IMP] stock_quant_manual_assign

### PR DESCRIPTION
**Pull request by kakurai8**
[1985](https://www.quartile.co/web#view_type=form&model=project.task&id=1985&active_id=1985&menu_id=)
- for update translation, clone from OCA

### CODE REVIEW

**Comment by kakurai8** [comment]
（取得元）
https://github.com/OCA/stock-logistics-warehouse/commits/12.0/stock_quant_manual_assign

A. OCAを正とする
B. 既存を正として/i18n/ja.poファイルのみ更新する
Aで問題ないかどうかの判断がついていないです
特に下記4. stock_quant_manual_assign/wizard/assign_manual_quants.pyの更新が不明

確認必要なファイル
stock_quant_manual_assign/manifest.py
stock_quant_manual_assign/i18n/ja.po
stock_quant_manual_assign/i18n/stock_quant_manual_assign.pot
stock_quant_manual_assign/wizard/assign_manual_quants.py

確認不要(と判断している)ファイル
5. 上記以外の/i18n/以下のファイル

File: stock_quant_manual_assign/wizard/assign_manual_quants.py [hunk 1]
```diff
@@ -60,24 +60,15 @@ def assign_quants(self):
...
     @api.model
     def default_get(self, fields):
         res = super(AssignManualQuants, self).default_get(fields)
         move = self.env['stock.move'].browse(self.env.context['active_id'])
-        available_quants = self._get_available_quants(move)
+        available_quants = self.env['stock.quant'].search([
+            ('location_id', 'child_of', move.location_id.id),
+            ('product_id', '=', move.product_id.id),
+            ('quantity', '>', 0),
+        ])
```
Comment (yostashiro) [inline stock_quant_manual_assign/wizard/assign_manual_quants.py:71]: This is a regression from just overriding the module with the latest one from the OCA repo.

ref. https://github.com/OCA/stock-logistics-warehouse/pull/1064

**Review by yostashiro — CHANGES_REQUESTED** [review]
When our module is diverted from what's in the OCA repo, we should maintain the translation separately in our repo and OCA repo. Although it's a bit of duplicated work, I can't think of a better way to handle this at this point.

**Comment by kakurai8** [comment]
This PR is closing.
I will recreate to adjust the translation only.

**Pull request closed without merging**